templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,18 @@
+Hello,
+
+We hope you are in good health and doing well.
+This email is in regards to PyCon India 2020 reviewer group. Pycon India is going virtual this year considering the current situations. We (conference organizers) take this as an opportunity to put more efforts in making this conference a great experience for attendees in other ways.
+
+We would like to put some more effort in bringing up some amazing talks to be presented this  Pycon 2020. We are creating a team of volunteers to review the talk proposals that should be presented in Pycon. Since this is a peer review process and driven by volunteers, we are looking for more volunteers to join us and help in making this a success. We believe you have great experience in Python and/or Python community and we would love if you can be a part of this group.
+
+Your role will be mainly to review the content of the talks submitted. You will be volunteering for a few talks you see fit as per your skills (we will help you with this) and provide your feedback if this should be a part of Pycon 2020. 
+
+If you think you can help us with this, please reply to the email confirming and we will add you to the group where we will discuss and plan/schedule sessions. 
+FAQs
+1. How much time do I need to dedicate ?
+Ans: For review, it will be around 10-15 hours over a period of 20 - 25 days
+
+2. How many meetings do I need to attend?
+Ans: For review, you will be required to attend 4-5 meetings for general discussion around review.
+
+Thank you.
```

Comment:
I feel we can end with a much warmer note 

- If you have any question feel free to ask or something 
- Looking forward to hearing back from you 
- A personal touch would go a long way.

Reply:
" Hoping to see you onboard with the team. Feel free to reach out for any queries " How does this sound. I want to keep it minimal.

Reply:
Minimal works :tada: